xml/admin_ceph_gateway.xml

```diff
@@ -288,12 +288,16 @@ for bucket in conn.get_all_buckets():
      you more about its command line options.
     </para>
     <para>
-     The package is included in the 'Public Cloud' module for &sle; 12 SP3.
+     The package is included in the 'Public Cloud' module for &sle; 12 SP3 and
+     &sle; 15 SP1.
      Before installing the package, you need to activate the module and refresh
      the software repository:
     </para>
 <screen>&prompt.root;SUSEConnect -p sle-module-public-cloud/12/x86_64
```

Comment:
Want to update for ARM here as well?

Reply:
We don't have thorough, consistent, documentation for ARM. Whilst I agree we should be including ARM in locations like this, I think that's a separate issue that we should be doing in one PR, rather than in small nits like this one.
I am happy to create another bug to ensure we go through the entire suite and look for ARM-related updates.

Reply:
Ok, let's do separately